amundsen_application/static/js/fixtures/metadata/tagsList.ts

```diff
@@ -0,0 +1,73 @@
+export const allTestTags = [
```

Comment:
nit: This is just tags, but be careful of copy pasting our staging dummy data and committing that code. It's best to commit our own dummy data always.